EnglishFilter/sections/whitelist.txt

```diff
@@ -394,6 +394,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/120990
 ! https://github.com/AdguardTeam/AdguardFilters/commit/1cabce688481281dccb756acf649eaf98f89be4c#r75718630
 ||fwcdn1.com/js/storyblock.js$badfilter
+! https://github.com/AdguardTeam/AdguardFilters/issues/121436
+@@||exoclick.com^$domain=exoclick.com
 ! https://github.com/AdguardTeam/CoreLibs/issues/1647
 @@||crazygames.com^$content
 ! https://github.com/AdguardTeam/AdguardFilters/issues/120784

```

Comment:
As I can see, we also have the rule in exclusions. It seems it's obsolete now.
I'm not sure that we have to unblock it due to https://forums.lanik.us/viewtopic.php?p=158751-exoclick-com.
@Alex-302 the last word is yours.

Reply:
Answered to user.